wp-content/themes/core/components/blocks/icon_grid/src/Icon_Grid.php

```diff
@@ -34,16 +34,19 @@ class Icon_Grid extends Block_Config implements Cta_Field {
 
 	public function add_block(): void {
 		$this->set_block( new Block( self::NAME, [
-			'title'       => __( 'Icon Grid', 'tribe' ),
-			'description' => __( 'A grid layout block with icon selections', 'tribe' ),
+			'title'       => esc_html__( 'Icon Grid', 'tribe' ),
+			'description' => esc_html__( 'A grid layout block with icon selections', 'tribe' ),
 			'icon'        => '<svg enable-background="new 0 0 146.3 106.3" version="1.1" viewBox="0 0 146.3 106.3" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:#16D690;}.st1{fill:#21A6CB;}.st2{fill:#008F8F;}</style><polygon class="st0" points="145.2 106.3 72.6 42.3 26.5 1.2 0 106.3"/><polygon class="st1" points="145.2 106.3 0 106.3 72.6 42.3 118.6 1.2"/><polygon class="st2" points="72.6 42.3 145.2 106.3 0 106.3"/></svg>',
-			'keywords'    => [ __( 'icon', 'tribe' ), __( 'grid', 'tribe' ) ],
+			'keywords'    => [ esc_html__( 'icon', 'tribe' ), esc_html__( 'grid', 'tribe' ) ],
 			'category'    => 'common', // core categories: common, formatting, layout, widgets, embed
 			'supports'    => [
 				'align'  => false,
 				'anchor' => true,
 				'html'   => false,
 			],
+			'model'       => Icon_Grid_Model::class,
```

Comment:
Note: this is where you'd define your MVC it's, of course we'd update the block generator to do this automatically.

I'm open to better ways for this as it's ideal, but it will work and provide flexibility if absolutely required.